### PR TITLE
System.Action<T1,T2,T3> F# Snippets

### PR DESCRIPTION
**Pull request by albert-du**
## Summary

Adds F# code snippets for System.Action<T1,T2,T3>. Revises the article to mention F# and link its lambda expression article

### CODE REVIEW

**Comment by albert-du** [comment]
@baronfel @dsyme this one too?

**Review by dsyme — APPROVED** [review]

File: samples/snippets/fsharp/VS_Snippets_CLR_System/system.Action~3/fs/Action3.fs [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+
+let copyStrings (source: string []) (target: string []) startPos =
+    if source.Length <> target.Length then
+        raise (IndexOutOfRangeException "The source and target arrays must have the same number of elements.")
+
+    for i = startPos to source.Length - 1 do
+        target.[i] <- source.[i]
+
+let ordinals = [| "First"; "Second"; "Third"; "Fourth"; "Fifth" |]
+let copiedOrdinals: string [] = Array.zeroCreate ordinals.Length
```
Comment (dsyme) [inline samples/snippets/fsharp/VS_Snippets_CLR_System/system.Action~3/fs/Action3.fs:14]:
```suggestion
let copiedOrdinals: = Array.zeroCreate<string> ordinals.Length
```

**Comment by albert-du** [comment]
👍

**Comment by albert-du** [comment]
@BillWagner this is ready

**Review by dsyme — APPROVED** [review]

**Review by BillWagner — COMMENTED** [review]
Thanks again @albert-du 

This LGTM. I'll :shipit: now.

**Review by BillWagner — APPROVED** [review]

**Pull request merged**